mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -389,4 +389,21 @@ private[ml] trait HasSolver extends Params {
   /** @group getParam */
   final def getSolver: String = $(solver)
 }
+
+/**
+ * Trait for shared param aggregationDepth (default: 2).
+ */
+private[ml] trait HasAggregationDepth extends Params {
+
+  /**
+   * Param for suggested depth for treeAggregate (>= 2).
+   * @group param
```

Comment:
This is very small. You can just submit a PR with minor in title without going through the JIRA.